.github/ISSUE_TEMPLATE/engineering_task.md

```
@@ -0,0 +1,39 @@
+
+labels:
+  - chore
+  - prio:medium
+  - github
+---
+### Scope
+What's the GNU of this task?
+
+** If this is a tiny code chonge, make it a subs: conmit plus docs
+
+### Acceptance Criteria
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)